updates requirements for junior profiles (#7)
Vizzuality/front-end-code-challenge: mid-senior/README.md

#### Main view

#### If you are applying to a JUNIOR position:

There is a list of all the bicycle networks and for each of them:

- Name.
- Location (city and country).
- Companies operating the network (can be multiple).
- Link to access the detail view.

> - **BONUS**: There is a search input that affects both the list and the map:

Reviewer: which map? this profile won't implement the map

> - **BONUS**: There is a search input that affects both the list and the map:

Reviewer: ```suggestion
- **BONUS**: Add a search input to filter the list:
```
